schemas/stsci.edu/asdf/0.1.0/asdf-schema.yaml

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+---
+$schema: "http://json-schema.org/draft-04/schema"
+id: "http://stsci.edu/schemas/asdf/0.1.0/asdf-schema"
+title:
+  ASDF schema
+description: |
+  A metaschema extending YAML Schema and JSON Schema to add support
+  for some ASDF-specific checks, related to nd-arrays.
```

Comment:
In `jwst_lib.models`, we want to be able to set up schema that says "This array must be two-dimensional, because I'm expecting a CCD image" etc.  We've actually had this feature for a long time over there, but now all of those underpinnings are being replaced with pyasdf, and I think it's general enough to include here.  But if it adds too much complexity, it could be a "private" feature of `jwst_lib.models`.